org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4viz/VizState.java

```diff
@@ -41,6 +41,8 @@
  * customization.
  * <p>
  * <b>Thread Safety:</b> Can be called only by the AWT event thread.
+ *
+ * @modified: Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
couple of changes, forbid projection over mutable sigs, and apply a default style to mutable elements (dashed)